minesweepers/uavcan_bridge/nodes/position_publisher.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python2
+"""
+Listens for UWB position on UAVCAN and forwards it on ROS
+"""
+
+import sys
+
+import uavcan
+import rospy
+
+import tf2_ros
+from geometry_msgs.msg import Point, TransformStamped
+
+def tf_from_position(x, y, z):
+    tf = TransformStamped()
+
+    tf.header.stamp = rospy.Time.now()
+    tf.header.frame_id = 'world'
+    tf.child_frame_id = 'uwb_link'
+    tf.transform.translation.x = x
+    tf.transform.translation.y = y
+    tf.transform.translation.z = z
+
+    return tf
+
+def main():
+    if len(sys.argv) < 3:
+        print("usage: position_publisher.py uavcan_dsdl_path can_interface")
+        return
+
+    dsdl_path, can_interface = sys.argv[1], sys.argv[2]
```

Comment:
use rosparam for this?

Reply:
yes, I still need to write a roslaunch and feed the values from rosparam. Stay tuned :tv: